mongoDB2/output_1.txt

```diff
@@ -0,0 +1,28 @@
+Question 1:
+1. How many records does each airline class have? Use $project to show result as { class: "Z", total: 999 } 
```

Comment:
I do not see that you are using projection + Answer is not  per example in question